src/tests/components/Navbar.test.js

```diff
@@ -4,18 +4,67 @@ import { BrowserRouter as Router } from 'react-router-dom'
 import { Navbar } from '../../components/navbar/Navbar'
 
 describe('Navbar', () => {
-  const props = { location: { pathname: '/users' } }
-  const homepage = (props) => mount(<Router><Navbar {...props} /></Router>)
+  let wrapper
+  const props = {
+    location: { pathname: '/users' },
+    cookies: { get: jest.fn() }
+  }
+
+  beforeEach(() => {
+    wrapper = mount(
+      <Router>
+        <Navbar {...props} />
+      </Router>
+    )
+  })
 
   describe('renders the Navbar component', () => {
     it('renders without errors', () => {
-      const wrapper = homepage(props)
       expect(wrapper.find('Navbar').length).toEqual(1)
     })
 
     it('renders 1 active Link element', () => {
-      const wrapper = homepage(props)
       expect(wrapper.find('.active').find('Link').length).toEqual(1)
     })
   })
+
+  describe('renders links based on if a user is signed in or not', () => {
+    it('renders a login link if a user in not signed in', () => {
+      const wrapper = mount(
+        <Router>
+          <Navbar {...props} />
+        </Router>
+      )
+      const loginLink = wrapper.find('a').filterWhere(item => {
+        return item.prop('href') === '/login'
+      })
+      expect(loginLink.text()).toEqual('Login')
+    })
+
+    it('renders a sign up link if a user in not signed in', () => {
+      const wrapper = mount(
+        <Router>
+          <Navbar {...props} />
+        </Router>
+      )
+      const loginLink = wrapper.find('a').filterWhere(item => {
+        return item.prop('href') === '/signup'
+      })
+      expect(loginLink.text()).toEqual('Sign up')
+    })
+
+    it('renders 9 menu items if a user is not signed in', () => {
```

Comment:
Can we instead check for the links to be present?
```
when('user is not logged in',() => {
   it('renders the signup and login menu items', () => {
   })
})
```
And also the reverse case

Reply:
I agree, I changed that as part of the `PR` adding the ability to log out, see https://github.com/AgileVentures/WebsiteOne-FE/pull/141/files